fetcher/windows_fetcher.go

```diff
@@ -160,6 +160,11 @@ func (wf *WindowsFetcherImpl) GetOracleDatabaseOratabEntries() []agentmodel.Orat
 	return marshal_oracle.Oratab(out)
 }
 
+// GetOracleDatabaseRunningDatabases get
+func (lf *WindowsFetcherImpl) GetOracleDatabaseRunningDatabases() []string {
+	return []string{}
```

Comment:
Maybe we should add a warn that it's not implemented?

Reply:
I have read the source code of win.ps1 and I have learned that windows don't have a concept of listed databases. GetOracleDatabaseOratab on windows return the list of running database processes (like ps -ef | grep pmon)